### ISSUE TITLE
Excessive logging?

### ISSUE THREAD

**Issue by jeff-h** [thompsonate/Shifty#96]
Shifty generates just over 16,000 log entries per day, for me at least. It's pretty much just the following, over and over:

```
[27/01/2020 at 3:13:46 PM]: sunset: 2020-01-27 08:07:28 +0000
[27/01/2020 at 3:13:46 PM]: sunrise: 2020-01-27 17:30:37 +0000
[27/01/2020 at 3:13:46 PM]: scheduled state: false
[27/01/2020 at 3:13:47 PM]: Responded to event: nightShiftDisableRuleDeactivated
```

I am wondering if it would be possible to reduce this? Perhaps it could be changed to only log when the content of the four lines changes? I'm not entirely certain how you'd do such a thing though :)

**Comment by choco** [comment]
Yeah I agree. We should probably put logging behind a flag so it can just be turned on when the user encounters a bug and we require additional info.

**Comment by thompsonate** [comment]
Agreed @choco. I'll plan on adding a flag in the next update.

16,000 lines per day is surprising though. At some point I'll try to look into how that can be reduced.
